requirements-dev.txt

```diff
@@ -18,7 +18,7 @@ isort==5.12.0
 # match the version with .pre-commit-config.yaml
 mypy==1.4.0
 types-PyYAML
-# 2.31 requires urlib3>2, which is incompatible with SkyPilot, IBM and
+# 2.31 requires urlib3>2, which is incompatible with IBM and
 # kubernetes packages, which require urllib3<2.
 types-requests<2.31
```

Comment:
possibly the `<2.31` here can now be removed as well?  but I held off on making that change, because I wasn't sure.